### PR TITLE
SC v. 4.8.0 release updates

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

Original PR (created from fork): https://github.com/saucelabs/sauce-docs/pull/1239

In preparation for the SC v. 4.8.0 release, @thehannahm updated the version lifecycle timeline section to remove no longer accessible versions, add 4.8.0, and bump 4.7.x into maintenance.

### Motivation and Context
We need to let customers know when versions are going to EOL, and this is an update to do that.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/secure-connections/sauce-connect/installation.md [hunk 1]
```diff
@@ -154,25 +154,38 @@ The launch of Sauce Connect 4.7.1 makes it the officially supported version of t
...
   </tr>
   <tr>
-   <td rowspan="2" >4.7
+   <td rowspan="2" >4.8
    </td>
-   <td>4.7.1
+   <td>4.8.0
    </td>
    <td>
     <a href="#download-latest-version">See latest version</a>
```
Comment (lysannep) [inline docs/secure-connections/sauce-connect/installation.md:162]:
```suggestion
    <a href="#downloading-sauce-connect-proxy">See latest version</a>
```

Comment (lysannep) [inline docs/secure-connections/sauce-connect/installation.md:162]: This changes impacts the "Downloading Sauce Connect Proxy" section in this same topic (see below), and also impacts other Sauce Connect topics that refer or link to version 4.7.1, including: **/secure-connections/sauce-connect/quickstart/#start-tunnel** and **/secure-connections/sauce-connect/setup-configuration/basic-setup/#basic-setup-with-a-test-script**

Screen Shot 2022-05-11 at 12 13 10 PM

Comment (thehannahm) [reply docs/secure-connections/sauce-connect/installation.md:162]: I've been told that these links and checksums auto populate with each new release, and that they didn't need to be updated.

Comment (thehannahm) [reply docs/secure-connections/sauce-connect/installation.md:162]: But you're right - I didn't see the code snippet, and that will need to be updated.

Comment (thehannahm) [reply docs/secure-connections/sauce-connect/installation.md:162]: I take that first comment back - apparently, the auto populate feature was removed from this page.

**Review by thehannahm — APPROVED** [review]
Looks good to me!

**Review by lysannep — APPROVED** [review]
Looks good. Thanks @thehannahm!

**Pull request merged**